### PR TITLE
Add ability to hide paginator

### PR DESCRIPTION
**Pull request by EricEhmke**
Added the ability to hide the paginator for single page documents.

### CODE REVIEW

**Review by anvithks — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ In your Hugo website place the following shortcode in any of the markdown pages.
 
 ### Parameters
 - **url (required)** : The relative location of the file.  
+- **hidePaginator (optional)**: Hides the paginator for single page documents
```
[the remark this answers is not in the document]
Comment (EricEhmke) [reply README.md:63]: @anvithkis 

Just saw this and can make these changes shortly

[the remark this answers is not in the document]
Comment (anvithks) [reply README.md:63]: @EricEhmke my apologies. i started the review and made these comments but missed to submit the review :facepalm: 
I reviewed the PR on the same day you submitted. whenever you get the time.
Thanks again.

[the remark this answers is not in the document]
Comment (EricEhmke) [reply README.md]: @anvithks all set. Let me know if you'd prefer a different format. Thanks!

**Review by anvithks — APPROVED** [review]

**Comment by anvithks** [comment]
@EricEhmke Thank you for your contribution!

**Pull request merged**